src/io-methods/azure.js

```diff
@@ -1,38 +1,52 @@
-const azure = require("azure-storage");
+const { BlobServiceClient } = require("@azure/storage-blob");
 let blobService;
 
-function getBlobService() {
-  blobService = blobService || azure.createBlobService();
+const AZURE_STORAGE_CONNECTION_STRING =
```

Comment:
This variable seems to be redundant/meaningless. Why not just use `process.env.AZURE_STORAGE_CONNECTION_STRING` instead?